ardere/aws.py

```diff
@@ -175,6 +180,98 @@ def request_instances(self, instances):
             ]
         )
 
+    def locate_metrics_container_ip(self):
+        """Locates the metrics container IP"""
+        response = self._ecs_client.list_container_instances(
+            cluster=self._ecs_name,
+            filter="task:group == service:metrics"
+        )
+        if not response["containerInstanceArns"]:
+            return None
+
+        container_arn = response["containerInstanceArns"][0]
+        response = self._ecs_client.describe_container_instances(
+            cluster=self._ecs_name,
+            containerInstances=[container_arn]
+        )
+
+        ec2_instance_id = response["containerInstances"][0]["ec2InstanceId"]
+        instance = self.boto.resource("ec2").Instance(ec2_instance_id)
+        return instance.public_ip_address
+
+    def locate_metrics_service(self):
+        """Locate and return the metrics service arn if any"""
+        response = self._ecs_client.describe_services(
+            cluster=self._ecs_name,
+            services=["metrics"]
+        )
+        if response["services"]:
+            return response["services"][0]
+        else:
+            return None
+
+    def create_influxdb_service(self, options):
+        # type: (Dict[str, Any]) -> Dict[str, Any]
+        """Creates an ECS service to run InfluxDB for metric reporting and
+        returns its info"""
+        logger.info("Creating InfluxDB service with options: {}".format(
+            options))
+
+        task_response = self._ecs_client.register_task_definition(
+            family=self.metrics_family_name(),
+            containerDefinitions=[
+                {
+                    "name": "metrics",
+                    "image": self.influxdb_container,
+                    "cpu": cpu_units_for_instance_type(
+                        options["instance_type"]),
+                    "memoryReservation": 256,
+                    "portMappings": [
+                        {"containerPort": 8086},
+                        {"containerPort": 8088}
+                    ],
+                    "logConfiguration": {
+                        "logDriver": "awslogs",
+                        "options": {
+                            "awslogs-group": self.container_log_group,
+                            "awslogs-region": "us-east-1",
```

Comment:
ardere is currently confined to us-east-1. There's a few ECS related reasons for this, like the fact that ECS seems to have clusters in a specific region, you can choose AZ's, but not regions. So at the moment, all of ardere is us-east-1 only.